src/tests/JIT/Regression/JitBlue/GitHub_26491/GitHub_26491.il

```diff
@@ -96,37 +100,68 @@
     IL_0009:  ret
   }
 
-  .method public hidebysig static int32 Main(string[] args) cil managed
+  .method public hidebysig static int32 MultipleReturns() cil managed
   {
     .custom instance void [xunit.core]Xunit.FactAttribute::.ctor() = (
         01 00 00 00
     )
-    .entrypoint
-    .maxstack  8
-    IL_0000:  ldarg.0
-    IL_0001:  ldstr      "MultipleReturns"
-    IL_0006:  call       int32 [System.Runtime]System.Array::IndexOf<string>(!!0[], !!0)
-    IL_000b:  ldc.i4.m1
-    IL_000c:  beq.s      IL_001b
-
     IL_000e:  ldc.i4     0x677b
     IL_0013:  call       valuetype GitHub_26491/ReturnType GitHub_26491::MultipleReturns(int32)
     IL_0018:  pop
-    IL_0019:  br.s       IL_0037
-
-    IL_001b:  ldarg.0
-    IL_001c:  ldstr      "SingleReturnSynchronized"
-    IL_0021:  call       int32 [System.Runtime]System.Array::IndexOf<string>(!!0[], !!0)
-    IL_0026:  ldc.i4.m1
-    IL_0027:  beq.s      IL_0031
+    IL_0037:  ldc.i4.s   100
```

Comment:
The `IL_0037` label definitely doesn't belong here - but I think it's okay and not worth trouble of re-doing ilasm/ildasm round trip to get them right.

Reply:
Hehe, thanks Egor for your feedback, as you can certainly see I just manually copied over the different sections of the previously main IL to the individual methods. This one is definitely a singleton but there are some tests that specify command-line arguments even though they don't refer to "base tests" to run, I'm investigating now what exactly happens there and it may have further implications on this aspect of il test refactoring.